### PR TITLE
PBI #521 - Sarine Profile – Love&Co QR - refine 

### PR DESCRIPTION
**Pull request by vlad-snrg**
https://sarinevso.visualstudio.com/DefaultCollection/SarineProfile/_workitems?_a=edit&id=521

Added template.js to HNNJ4XZQ9X and SAWM92EHK2 configurators to enable click event handler on summary experience  text

[PBI-#521-Love-refine-pull-request.zip](https://github.com/SarineTechnologies/sarine.viewer.templates.dynamic/files/218589/PBI-.521-Love-refine-pull-request.zip)

### CODE REVIEW

File: app/dist/configurations/HNNJ4XZQ9X/scripts/template.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+(function ($, tplUtils) {
+    document.querySelector(".summary__heading").addEventListener("click", function() {
+        tplUtils.fire(document.querySelector(".summary__story:not(.slick-cloned)"), "click");
+    });
+})(window.jQuery, window.tplUtils);
```
Comment (adica) [inline app/dist/configurations/HNNJ4XZQ9X/scripts/template.js]: @vlad-snrg 
if you inject jQuery - why do you use " document.querySelector "?

**Comment by vlad-snrg** [comment]
We did it in order to make it work on the iPhones.
Jquery.click() is not working correctly on iPhone, so we use document.querySelector to enable addEventListener method

File: app/dist/configurations/HNNJ4XZQ9X/scripts/template.js (hunk 1 above)
Comment (vlad-snrg) [inline app/dist/configurations/HNNJ4XZQ9X/scripts/template.js]: We did it in order to make it work on the iPhones.
Jquery.click() is not working correctly on iPhone, so we use document.querySelector to enable addEventListener method

Comment (adica) [inline app/dist/configurations/HNNJ4XZQ9X/scripts/template.js]: @vlad-snrg 
Ok - no problem (Just no need to inject jQuery if you don't use it).
The rest of the pull request look fine.
Thanks

**Pull request merged**